editor/src/components/inspector/sections/layout-section/layout-system-subsection/split-chained-number-input.tsx

```diff
@@ -214,6 +222,70 @@ const handleSplitChainedEvent =
       unsetProperty(element, longhand.L),
     ]
 
+    function actionsOrUnset(
+      actions: EditorAction[],
+      unset: boolean,
+      unsetPaths: PropertyPath[],
+    ): EditorAction[] {
+      if (unset) {
+        return unsetPaths.map((path) => unsetProperty(element, path))
+      }
+      return actions
+    }
+
+    function shouldUnset(): boolean {
+      switch (e.type) {
+        case 'one-value':
+          return e.value == null
+        case 'two-value':
+          const otherDirections = useShorthand
+            ? e.value.type === 'V'
+              ? [aggregates.horizontal]
+              : [aggregates.vertical]
+            : e.value.type === 'V'
+            ? [aggregates.top, aggregates.bottom]
+            : [aggregates.right, aggregates.left]
```

Comment:
This nested ternary threw me off 😬 But do we even need the nesting here? Wouldn't `[aggregates.top, aggregates.bottom]` both be zero if `[aggregates.vertical]` is zero?